### PR TITLE
OSSM-4982: SM 3.0 Installing the Service Mesh Operator

### PR DESCRIPTION
**Pull request by rh-tokeefe**
Version(s):  Service Mesh 3.0

Issue: https://issues.redhat.com/browse/OSSM-4982

Link to docs preview: https://75037--ocpdocs-pr.netlify.app/openshift-service-mesh/latest/install/ossm-installing-openshift-service-mesh.html#ossm-installing-operator

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by fjglira** [comment]
Hey @rh-tokeefe I know that is not related to your change but reading the introduction to the install service mesh section I saw this:
```
Installing OpenShift Service Mesh consists of three main tasks: installing the OpenShift Operator, deploying the Istio control plane, and customizing the Istio configuration. Then, you can also choose to install the sample bookinfo application to push data through the mesh and explore mesh functionality.
```
Here: `deploying the Istio control plane` we need to add `deploying the Istio control plane, deploying istio cni....xxxx` this is becase now cni is deployed separated from the istio control plane resource.

**Comment by fjglira** [comment]
Also here:
`You have deployed a cluster on OpenShift Container Platform 4.13 or later.` I think we talked about putting here the latest GA, now is 4.15

`Search on Sail.` I think is more clear to use `Search for Sail.`

File: modules/ossm-installing-operator.adoc [hunk 1]
```diff
@@ -15,11 +15,9 @@ You have access to the OpenShift CLI (oc).
...
 
+ifdef::openshift-rosa[]
 . Scroll, or type the keyword *Serverless* into the *Filter by keyword* box to find the {ServerlessOperatorName}.
-
-. Navigate to the OperatorHub.
-
-. Click *Operator* -> *Operator Hub*.
+endif::openshift-rosa[]
 
 . Search on Sail.
```
Comment (fjglira) [inline modules/ossm-installing-operator.adoc:22]: Maybe `Search for Sail.` sounds clearer

Comment (rh-tokeefe) [reply modules/ossm-installing-operator.adoc:22]: agree, done.

**Review by fjglira — APPROVED** [review]
lgtm

**Comment by fjglira** [comment]
I'll wait until the docs are generated again, but from PR as we talked everything looks good

**Review by fjglira — APPROVED** [review]
lgtm

File: modules/ossm-installing-operator.adoc [hunk 2]
```diff
@@ -5,23 +5,19 @@
...
 You are logged in to the {ocp-product-title} web console as a user with the cluster-admin role.
 
-You have access to the OpenShift CLI (oc).
-
 .Procedure
 
 . In the {ocp-product-title} web console, navigate to the *Operators* -> *OperatorHub* page.
 
+ifdef::openshift-rosa[]
 . Scroll, or type the keyword *Serverless* into the *Filter by keyword* box to find the {ServerlessOperatorName}.
```
Comment (dgn) [inline modules/ossm-installing-operator.adoc:17]: I'm not sure why Serverless shows up here?

**Review by dgn — APPROVED** [review]
Other than Serverless mention lgtm

File: modules/ossm-installing-operator.adoc (hunk 2 above)
Comment (rh-tokeefe) [reply modules/ossm-installing-operator.adoc:17]: @dgn It's conditionalized and only appears when serverless doc builds. This comment was in the source file, so I was thinking it was added in the 2.x content and will likely be needed for serverless in 3.0. 

Do you think it should be deleted?

Comment (rh-tokeefe) [reply modules/ossm-installing-operator.adoc]: Deleted

**Review by gabriel-rh — APPROVED** [review]
LGTM

**Pull request merged**